### PR TITLE
Add phenotype-extractor service

### PR DESCRIPTION
**Pull request by adityaekawade**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -368,13 +368,21 @@ router.post('/clinvarCountsForGene', async (ctx) => {
 // genepanel endpoints
 //
 router.get('/clinphen', async (ctx) => {
-
+
```
Comment (adityaekawade) [inline src/index.js]: I had added a console.log statement here to test. Looks like an extra space got added in the process.

File: src/index.js [hunk 2]
```diff
@@ -368,13 +368,21 @@ router.post('/clinvarCountsForGene', async (ctx) => {
...
 router.get('/clinphen', async (ctx) => {
-
+ 
   const args = [ctx.query.notes];
 
   await handle(ctx, 'clinphen.sh', args);
 });
 
 
+router.get('/phenotype-extractor', async (ctx) => {
```
Comment (anderspitman) [inline src/index.js]:
```suggestion
router.get('/phenotypeExtractor', async (ctx) => { 
```

File: tools/phenotypeExtractor/phenotype-extractor.def [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Bootstrap: library
+From: alpine:3.9
+
+%post
+
+    apk update
+
+    #apk add build-base git zlib-dev bzip2-dev xz-dev xz-libs curl-dev
```
Comment (anderspitman) [inline tools/phenotypeExtractor/phenotype-extractor.def]:
```suggestion
```

Comment (anderspitman) [reply tools/phenotypeExtractor/phenotype-extractor.def]: Remove comment line

File: tools/phenotypeExtractor/phenotype-extractor.def [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+    python -m nltk.downloader -d /usr/share/nltk_data wordnet
+
+    git clone https://github.com/adityaekawade/Phenotype-extractor.git
+
+    cd Phenotype-extractor
+
+    npm install
+
+    npm link
```
Comment (anderspitman) [inline tools/phenotypeExtractor/phenotype-extractor.def]: Shouldn't be necessary since you're running cli.js directly
```suggestion
```

**Comment by anderspitman** [comment]
@adityaekawade added a couple more items that I missed before. Sorry, I should have looked closer. Also, can you add phenotypeExtractor to the list of tools at the top of the makefile? It uses that list to download the tools from S3 when you run make. I've already uploaded the phenotypeExtractor binary to S3.

**Comment by adityaekawade** [comment]
> @adityaekawade added a couple more items that I missed before. Sorry, I should have looked closer. Also, can you add phenotypeExtractor to the list of tools at the top of the makefile? It uses that list to download the tools from S3 when you run make. I've already uploaded the phenotypeExtractor binary to S3.

@anderspitman , Uodated

**Comment by anderspitman** [comment]
@adityaekawade it looks like there are merge conflicts now. Did you originally fork from dev or master?

**Comment by anderspitman** [comment]
Ah I see. I think your branch is behind master. You'll need to merge the latest and push to your branch.

**Comment by adityaekawade** [comment]
> @adityaekawade it looks like there are merge conflicts now. Did you originally fork from dev or master?

From master branch

**Comment by adityaekawade** [comment]
@anderspitman , I resolved the conflicts, and it is now no longer behind the master branch.

**Comment by anderspitman** [comment]
@adit Makefile looks good now. Can you fix the other 2 changes from the latest review? Basically the naming needs to be consistent or it can't find the script that gets downloaded. Sorry, this is hard for you to test since I'm the one who uploads to S3. I'll try to think of a better way to do it in the future.

**Pull request merged**